packages/blade/src/components/Card/_decisions/decisions.md

```diff
@@ -0,0 +1,265 @@
+
+- Never use a primary/secondary action in the header, footer are meant for such actions.
+  - This means it's not recommended to change variant of IconButton
+
+**Footer Leading Section**
+
+- `Title` & `Subtitle`
+  - Size & text variant cannot be changed
+
+**Footer Trailing Sections**
+
+- Only two action buttons can be present `primary` or `secondary`
+- Users can totally omit action buttons
+- They can also use either variants of buttons `primary` or `secondary`
+
+## API
+
+With the above constraints we can propose the Card API as follows: 
+
+Sample usage:
+
+```jsx
+<Card backgroundLevel={1}>
+  <Card.Header 
+    title="Payments Links" 
+    subtitle="Share payment link via an email, SMS, messenger, chatbot etc." 
+    titlePrefix={<DollarIcon size="xl" />}
+    titleSuffix={<Counter amount={20} />}
+    trailingVisual={<Badge variant="positive">NEW</Badge>}
+  />
+  <Card.Body>
+    Card Body Content
+  </Card.Body>
+  <Card.Footer
+    title="Card Footer Title" 
+    subtitle="Card footer subtitle" 
+    actions={{
+      primaryAction: { text: 'Know more', onClick: () => {} },
+      secondaryAction: { text: 'Read Docs', onClick: () => {} },
+    }} 
+  />
+</Card>
+```
+
+
+### `Card` API
+
+
+| Prop              | Type   | Default | Description                                                                            | Required |
+| ----------------- | ------ | ------- | -------------------------------------------------------------------------------------- | -------- |
+| `backgroundLevel` | `2, 3` | `2`     | Surface level of the card background color, use this based on where the card is placed |          |
+
+
+### `Card.Header` API
+
+
+| Prop             | Type              | Default     | Description                                              | Required |
+| ---------------- | ----------------- | ----------- | -------------------------------------------------------- | -------- |
+| `title`          | `string`          | `undefined` | Title of the Card                                        | ✅        |
+| `subtitle`       | `string`          | `undefined` | Subtitle of the Card                                     |          |
+| `titlePrefix`    | `React.ReactNode` | `undefined` | Prefix element placed before title text (eg: Icon)       |          |
+| `titleSuffix`    | `React.ReactNode` | `undefined` | Suffix element placed after title text (eg: Counter)     |          |
+| `trailingVisual` | `React.ReactNode` | `undefined` | Trailing visual element placed on right side of the card |          |
+
+### `Card.Footer` API
+
+| Prop       | Type                                     | Default     | Description                                | Required |
+| ---------- | ---------------------------------------- | ----------- | ------------------------------------------ | -------- |
+| `title`    | `string`                                 | `undefined` | Title of the Card                          | ✅        |
```

Comment:
We have use-cases where there are only action buttons in the footer. Let's make title optional?

<img width="571" alt="Screenshot 2022-11-30 at 3 54 33 PM" src="https://user-images.githubusercontent.com/24487274/204771737-b7f3ff79-6729-4eab-9e89-5e7cf38ff7e5.png">

```suggestion
| `title`    | `string`                                 | `undefined` | Title of the Card                          |        |
```

Reply:
Good catch!

Reply:
Resolved with the new API.